pkg/cost/prometheus.go

```
@@ -0,0 +1,280 @@
+
+	queryLog := log.WithField("query", query)
+
+	var results []PrometheusResult
+	err = retry.Do(
+		func() error {
+			resp, err := c.httpClient.Get(u.String())
+			if err != nil {
+				queryLog.Warnf("request failed (will retry): %v", err)
+				return err
+			}
+			defer resp.Body.Close()
```

golangci-lint (2.12.2)

[error] Error return value of `resp.Body.Close` is not checked

(errcheck)